_topic_map.yml

```
@@ -2913,17 +2921,21 @@ Topics:
     # Metering
   - Name: Using metering with OpenShift Serverless
     File: serverless-metering
-# Architecture
-- Name: Architecture
-  Dir: architecture
-  Topics:
-  - Name: Knative Serving
-    File: serverless-serving-architecture
-  - Name: Knative Eventing
-    File: serverless-event-architecture
-# HA
-- Name: High availability on OpenShift Serverless
-  File: serverless-HA
+  # HA
+  - Name: High availability on OpenShift Serverless
+    File: serverless-ha
+# Service mesh integration
+- Name: Service Mesh integration
+  Dir: service_mesh
+  Topics:
+  - Name: Integrating Service Mesh and OpenShift Serverless with Kourier enabled
+    File: serverless-ossm-with-kourier
+  - Name: Configuring TLS for a custom domain using Service Mesh and Kourier
+    File: serverless-ossm-tls-with-kourier
+  - Name: Integrating Service Mesh and OpenShift Serverless without Kourier
+    File: serverless-ossm-without-kourier
+  - Name: Using custom domains for Knative services with Service Mesh
```

```suggestion
  - Name: Configuration custom domains using Service Mesh and Kourier
```

This chapter does not work without Kourier so let's add "and Kourier".

---

YES thank you I was curious about this! If they are just using Service Mesh would they use `DomainMapping` objects instead?

---

Yes, that's correct. They can use `DomainMapping`.

---

Actually ServiceMesh w/ Kourier can also use `DomainMapping` but w/ Kourier + DomainMapping does not support TLS :disappointed:

---

No problem, I'll be sure to make a note of that somewhere too since I think it's worth mentioning